data/json/items/tools.json

```diff
@@ -6634,7 +6634,24 @@
     "color": "brown",
     "techniques": "WBLOCK_1",
     "qualities": [ [ "CUT", 1 ], [ "COOK", 1 ] ],
-    "flags": [ "SPEAR", "REACH_ATTACK", "NONCONDUCTIVE", "FRAGILE_MELEE" ]
+    "flags": [ "SPEAR", "REACH_ATTACK", "NONCONDUCTIVE", "FRAGILE_MELEE", "SHEATH_SPEAR" ]
+  },
+  {
+    "id": "spear_homemade_halfpike",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "homemade halfpike",
+    "description": "A short do-it-yourself spear made of a smooth wooden shaft with a shaped metal shape seated and bound into place at its tip. Its functional grip and decent construction makes it a usable, if not effective, weapon.",
+    "weight": 1814,
+    "volume": 6,
+    "price": 1400,
+	"to_hit": 1,
+    "bashing": 2,
+    "cutting": 19,
```

Comment:
I thought pike is made for stabbing, not cutting?

Reply:
It does piercing damage in-game. Not certain why; there doesn't seem to be a "piercing" attribute, but the `SPEAR` flag appears to.